DashAI/back/config_object.py

```diff
@@ -16,3 +17,7 @@ def get_schema(cls) -> dict:
             Dictionary representing the Json Schema of the component.
         """
         return cls.SCHEMA.model_json_schema()
+
+    def validate_and_transform(self, raw_schema: dict) -> dict:
```

Comment:
Pls document this function!

Reply:
It also adds a note of the exception or error that should be raised when a data set is incorrectly validated against your schema.